Examples/Algorithms/TrackFinding/src/TrackParamsEstimationAlgorithm.cpp

```diff
@@ -79,8 +79,15 @@ ActsExamples::TrackParamsEstimationAlgorithm::createSeeds(
 
   std::unordered_map<Index, const SimSpacePoint*> spMap;
 
-  for (const SimSpacePoint& i : spacePoints) {
-    spMap.emplace(i.measurementIndex(), &i);
+  for (const SimSpacePoint& sp : spacePoints) {
+    if (sp.sourceLinks().size() == 0) {
+      ACTS_WARNING("Missing soucelink in space point");
+      continue;
+    }
+    const auto slink =
+        static_cast<const IndexSourceLink&>(*(sp.sourceLinks()[0]));
+    spMap.emplace(slink.index(),
+                  &sp);  // The second measurement is ignored for strips
```

Comment:
Should we really ignore this ? It should link to both, shouldn't it?

Reply:
Fixed. Both indices are now added.